ntbs-service/Pages/Notifications/Overview.cshtml.cs

```diff
@@ -93,6 +95,28 @@ private void CalculateTreatmentOutcomes()
             }
         }
 
+        // We only want to add a 'pseudo-event' which is displayed but isn't actually tied to the notification/outcomes
+        private void AddDenotificationEventIfDenotified()
+        {
+            if (Notification.NotificationStatus == NotificationStatus.Denotified)
+            {
+                var denotificationEvent = new TreatmentEvent
+                {
+                    EventDate = Notification.DenotificationDetails.DateOfDenotification,
+                    TreatmentEventType = TreatmentEventType.Denotification
+                };
+                if (TreatmentPeriods.Any())
+                {
+                    TreatmentPeriods.Last().TreatmentEvents.Add(denotificationEvent);
+                    TreatmentPeriods.Last().TreatmentEvents.OrderForEpisodes();
+                }
+                else
+                {
+                    TreatmentPeriods.Add(TreatmentPeriod.CreateTreatmentPeriod(1, denotificationEvent));

```

Comment:
Is it actually possible for us to end up in this case?

Reply:
No, after investigation it isn't - as we always have a treatment start or diagnosis made event.